data/seeds/0001_create_tracks_seeding.js

```diff
@@ -0,0 +1,8 @@
+// what here?????????
+exports.seed = function(knex, Promise) {
+  return knex('dishes').insert([
+    {name: 'WEB'},
```

Comment:
Same here, copy and paste may be faster but if you are not careful, it has this consequences. I highly recommend using old projects as reference, but I don't recommend copy and paste, even if you are referencing old code, you should write it down by yourself, this way you'll get a better understanding of what you are doing, you'll get real practice and you'll get familiar with the job, plus if you are debugging an error you are more likely to know where is it coming from since you wrote the whole thing, when copy and pasting its hard to know where errors are coming from. This seeding won't give you any error because the dish table has a name property, but I doubt you can find any plate out there named WED, Android or IOS, LOL